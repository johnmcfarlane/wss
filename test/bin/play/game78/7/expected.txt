 49  1,11 - ASP
 48  6,11 - SONJA
 46  5,2  | POOJAS
 44  5,3  | JONAS
 39  3,7  | PODOS
 36  2,8  - SAP
 33  4,8  | JATO
 30  3,6  | ANODOS
 30  6,11 - SNOOP
 30  6,11 - SPOON
 28  5,5  | JA
 27  5,3  | JA
 27  5,3  | JO
 27  6,11 - SNAP
 27  6,11 - SNOP
 27  6,11 - SPAN
 26  5,1  | JASON
 25  5,2  | JOAN
 25  5,2  | JOON
 25  5,3  | POOA
 25  8,7  | SAN
 25  8,8  | SNAP
 25  8,8  | SNOP
 24  3,7  | JODO
 24  5,4  - OWENIA
 24  2,8  - SAN
 24  6,11 - SAPO
 24  6,11 - SOAP
 24  7,1  - SQUIZ
 23  5,4  | OPA
 22  5,1  | NAPOO
 22  2,6  | PANELS
 22  5,2  | SNAP
 22  5,1  | SNOOP
 22  5,2  | SNOP
 22  2,11 - SO
 22  5,2  | SPAN
 21  8,8  | ANS
 21  3,9  | DOP
 21  5,3  | NAP
 21  5,2  | NOOP
 21  5,3  | PAN
 21  5,5  | PAS
 21  5,3  | PON
 20  8,8  | AN
 20  5,2  | ASOP
 20  2,6  | PANEL
 20  5,2  | SOAP
 20  5,1  | SPOON
 19  3,8  - AN
 19  5,4  | ONA
 19  5,2  | PAON
 19  5,2  | POON
 19  5,3  | SAP
 19  5,3  | SOP
 18  3,8  | ADON
 18  8,9  | NAP
 18  2,8  - SAO
 18  6,11 - SOON
 17  5,4  | OP
 17  5,1  | SAPO
 16 10,1  | IODO
 16  5,2  | OPA
 16  4,7  | OPATA
 16  5,5  | PA
 16  5,2  | SOON
 16  5,2  | SPA
 15  3,8  | ADOS
 15  3,9  | DON
 15  8,9  | NOA
 15  5,3  | NOO
 15  8,9  | NOO
 15  8,9  | NOS
 15  5,3  | PA
 15  3,7  - PANELS
 15  4,8  | PATA
 15  4,8  | PATO
 15  4,8  | PATS
 15  5,3  | PO
 15  5,3  | SAN
 15  5,3  | SON
 15 11,3  | SONJA
 15  4,7  | SPAT
 14  2,6  | ANSEL
 14  7,8  - APA
 14  2,7  | JOEL
 14  4,7  - JOEL
 14  8,9  | NA
 14  8,9  | NO
 14  3,7  - PANEL
 14  5,1  | PASO
 14  4,8  | PAT
 14  5,7  - PELANOS
 14  5,1  | POOA
 13  4,5  - AJAR
 13  5,4  | AN
 13  3,8  - AS
 13  6,7  - ELAPS
 13  6,7  - ELOPS
 13  5,4  | ON
 13  5,3  | SAO
 13  3,6  | SPADO
 12  3,9  | DOA
 12  3,9  | DOO
 12  3,9  | DOS
 12  5,5  - JAR
 12  5,5  | NA
 12  5,1  | NASO
 12  5,2  | ONA
 12  5,2  | ONO
 12  5,7  - PELON
 12 11,1  | ZAS
 11  5,6  | AS
 11  5,3  | NA
 11  5,3  | NO
 11  5,2  | NOA
 11  5,2  | NOO
 11  5,4  | OO
 11  3,6  | PONDO
 11  4,6  | SONATA
 11  1,9  - fEDAYEEN
 10  7,8  - ANOAS
 10  5,2  | SAO
 10 11,3  | SNOOPS
 10 11,3  | SPOONS
  9  7,8  - ANAS
  9  4,7  | ANAT
  9  7,8  - ANOA
  9  7,8  - ANSA
  9  7,8  - ASAP
  9  7,8  - ASOP
  9  9,5  - CAP
  9  9,5  - COP
  9  9,5  - CSP
  9  6,7  - ELANS
  9  4,8  | NATO
  9  5,3  | OO
  9  6,11 - SAP
  9 11,3  | SNAPS
  9 11,3  | SNOOP
  9  5,3  | SO
  9  6,11 - SOP
  9  6,11 - SPA
  9 11,3  | SPANS
  9 11,3  | SPOON
  8  7,8  - ANA
  8  7,8  - ANS
  8  3,7  - ANSEL
  8  1,11 - AS
  8  7,8  - ASP
  8  6,7  - ELAN
  8  6,7  - ELON
  8  4,6  | NASAT
  8  4,8  | NAT
  8  1,11 - OS
  8 11,3  | SNAP
  8 11,3  | SNOP
  8 11,3  | SOAPS
  8 11,3  | SPAN
  7  7,8  - AN
  7  3,5  - ANSAR
  7  4,5  - APAR
  7  9,5  - CAN
  7  9,5  - CON
  7  2,7  | NOELS
  7  4,7  - NOELS
  7  1,8  - OS
  7  4,5  - PAAR
  7  6,11 - SAN
  7 11,3  | SAPO
  7 11,3  | SAPS
  7 11,3  | SOAP
  7  6,11 - SON
  7  3,5  - SONAR
  7 11,3  | SOPS
  7  4,5  - SPAR
  7 11,3  | SPAS
  7  1,5  | SPOOf
  6  5,6  - AMuS
  6  5,4  - AWE
  6  9,5  - COO
  6  9,5  - COS
  6  6,7  - ELSA
  6  6,6  - MuN
  6  2,7  | NOEL
  6  4,7  - NOEL
  6  4,8  | OATS
  6  5,4  - OWE
  6  5,5  - PAR
  6  1,6  | POOf
  6  3,7  | SADO
  6  6,11 - SAO
  6 11,3  | SAP
  6  4,8  | SATA
  6 11,3  | SOP
  6 11,3  | SPA
  6 11,3  | SSP
  6  5,9  | YON
  5  7,8  - AAS
  5  5,6  - AMu
  5 10,3  | DO
  5  9,4  - IO
  5  6,6  - MuS
  5  4,8  | OAT
  5 11,3  | SANS
  5  4,8  | SAT
  5  6,11 - SO
  5 11,3  | SONS
  5 11,3  | SOON
  4  7,8  - AA
  4  3,8  | ADO
  4  7,8  - AS
  4  6,7  - ELS
  4  2,9  | ELS
  4  5,5  - NAR
  4  4,5  - OSAR
  4 11,3  | SAN
  4  5,7  - SEL
  4  2,8  | SEL
  4  4,5  - SOAR
  4 11,3  | SON
  4 11,3  | SOSO
  3  6,9  | ESS
  3  1,7  | NAf
  3  5,5  - OAR
  3 11,3  | SAO
  3 11,3  | SOS
  2  1,7  | OAf
  2  1,7  | OOf
  2 11,3  | SO
  1  1,8  | Of
