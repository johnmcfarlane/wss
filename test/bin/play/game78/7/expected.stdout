 36  2,8  - SAP
 33  4,8  | JATO
 30  6,11 - SNOOP
 30  6,11 - SPOON
 28  5,5  | JA
 27  5,3  | JA
 27  5,3  | JO
 27  6,11 - SNAP
 27  6,11 - SPAN
 25  8,8  | SNAP
 24  6,11 - SOAP
 22  2,6  | PANELS
 22  5,2  | SNAP
 22  5,1  | SNOOP
 21  5,3  | NAP
 21  5,5  | PAS
 20  8,8  | AN
 20  2,6  | PANEL
 20  5,2  | SOAP
 19  5,3  | POO
 19  5,3  | SAP
 19  5,3  | SOP
 18  8,9  | NAP
 18  6,11 - SOON
 17  5,4  | OP
 16  5,2  | APO
 16  5,5  | PA
 16  5,2  | SPA
 15  3,8  | ADOS
 15  7,8  - APOS
 15  3,9  | DON
 15  5,3  | NOO
 15  8,9  | NOO
 15  8,9  | NOS
 15  5,3  | PA
 15  3,7  - PANELS
 15  4,8  | PATS
 15  5,3  | PO
 15  4,7  | SPAT
 14  7,8  - APO
 14  8,9  | NA
 14  8,9  | NO
 14  3,7  - PANEL
 14  4,8  | PAT
 13  4,5  - AJAR
 13  3,8  - AS
 13  5,2  | POO
 13  3,6  | SPADO
 12  3,9  | DOS
 12  5,5  - JAR
 12  5,2  | ONO
 12  5,7  - PELON
 12 11,1  | ZAS
 11  5,6  | AS
 11  5,3  | NA
 11  5,3  | NO
 11  5,2  | NOO
 11  4,6  | SONATA
 11  1,9  - fEDAYEEN
 10  7,8  - ANOAS
 10 11,3  | SNOOPS
 10 11,3  | SPOONS
  9  7,8  - ANAS
  9  7,8  - ANOA
  9  7,8  - ANSA
  9  9,5  - CAP
  9  9,5  - COP
  9  6,7  - ELANS
  9  6,11 - SAP
  9 11,3  | SNAPS
  9 11,3  | SNOOP
  9  5,3  | SO
  9  6,11 - SOP
  9  6,11 - SPA
  9 11,3  | SPANS
  9 11,3  | SPOON
  8  7,8  - ANA
  8  1,11 - AS
  8  7,8  - ASP
  8  6,7  - ELAN
  8  1,11 - OS
  8 11,3  | SNAP
  8 11,3  | SOAPS
  8 11,3  | SPAN
  7  7,8  - AN
  7  9,5  - CAN
  7  9,5  - CON
  7  2,7  | NOELS
  7  4,7  - NOELS
  7  1,8  - OS
  7 11,3  | SAPS
  7 11,3  | SOAP
  7  6,11 - SON
  7  3,5  - SONAR
  7 11,3  | SOPS
  7  4,5  - SPAR
  7 11,3  | SPAS
  7  1,5  | SPOOf
  6  5,6  - AMuS
  6  5,4  - AWE
  6  9,5  - COO
  6  9,5  - COS
  6  6,6  - MuN
  6  2,7  | NOEL
  6  4,7  - NOEL
  6  4,8  | OATS
  6  5,4  - OWE
  6  5,5  - PAR
  6  1,6  | POOf
  6 11,3  | SAP
  6 11,3  | SOP
  6 11,3  | SPA
  6  5,9  | YON
  5  7,8  - AAS
  5  5,6  - AMu
  5  6,6  - MuS
  5  4,8  | OAT
  5 11,3  | SANS
  5  4,8  | SAT
  5  6,11 - SO
  5 11,3  | SONS
  5 11,3  | SOON
  4  7,8  - AA
  4  3,8  | ADO
  4  7,8  - AS
  4  6,7  - ELS
  4  2,9  | ELS
  4  4,5  - OSAR
  4  5,7  - SEL
  4  2,8  | SEL
  4  4,5  - SOAR
  4 11,3  | SON
  3  6,9  | ESS
  3  5,5  - OAR
  3 11,3  | SOS
  2  1,7  | OAf
  2 11,3  | SO
  1  1,8  | Of
